tests/manual/placeholder.html

```diff
@@ -1 +1,5 @@
+<style type="text/css">
+	.myUi { display: inline-block; width: 6px; height: 6px; position: relative; top: -16px; background: #AA0000; pointer-events: none; }
```

Comment:
I needed to give this marker a shape. With `display: inline` it would be empty. I could create a UI element that is inline and has `inline-block` or `position: absolute` element inside, though (but I didn't check if it will work).